runtime/protocol/http-client/jvm/test/aws/smithy/kotlin/runtime/http/interceptors/requestcompression/GzipCompressionTest.kt

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package aws.smithy.kotlin.runtime.http.interceptors.requestcompression
```

Comment:
**fix**: These tests appear to be testing the `GzipSdkSource` and `GzipByteReadChannel` abstractions directly, they belong in `runtime-core` tests closer to where those types are declared.